package.json

```diff
@@ -26,6 +26,10 @@
   },
   "scripts": {
     "build": "pnpm --recursive --if-present build",
-    "lint": "pnpm --recursive --if-present lint"
+    "lint": "pnpm --recursive --if-present lint",
+    "prepare": "husky install && bash -c 'chmod ug+x .husky/*'"
```

Comment:
This seems to be used for the prepare script to fail silently, if husky is not installed. This can be used to disable husky for example in ci / docker / prod context (see https://typicode.github.io/husky/guide.html under "Disable husky in CI/Docker/Prod".

With the new commit we should no longer need the 'chmod'. If creating a husky hook with their cli they automatically set it as executable (https://typicode.github.io/husky/troubleshooting.html) . Seems like I created the 'prepare_commit_msg' file manually and therefore it wasn`t executable.

Would be nice if you could check this. With my unix laptop it worked

Reply:
The current version still fails for me, but I think it has more to do with the location of `add-commit-message.js` that can not be found. Also, node is not available in some environments per default, so it fails even when the js file is moved to the correct location.